src/renderer/components/Recent.tsx

```diff
@@ -54,14 +54,10 @@ const RecentList = () => {
           const { connectionConfig } = connection;
 
           let connectionString = '';
-          if (connectionConfig.config === 'manual') {
-            connectionString =
-              `${connection.type}://${connectionConfig.username}:` +
-              `asdf` +
-              `@${connectionConfig.address}:${connectionConfig.port}`;
-          } else if (connectionConfig.config === 'string') {
-            connectionString = connectionConfig.connectionString;
-          }
+          connectionString =
+            `${connection.type}://${connectionConfig.username}:` +
+            `asdf` +
```

Comment:
Maybe we can replace the password with four asterisks? That's how MongoDB Compass does it actually
```suggestion
            `****` +
```